### DISCUSSION TITLE
How to put this code into my WordPress website

### DISCUSSION THREAD

**Discussion by HamzaAshfaqSKG** [dmuy/duDatepicker#44] [Q&A]
hi, everyone, i want to put this code into my WordPress website and want to open using contact form 7... please help me ASAP

**Comment by dmuy** [comment]
Hi @HamzaAshfaqSKG,

Sorry for the late reply. Also, I'm not that familiar of how to use plugins on WordPress, but right now there's no WordPress version of this library (if this needs to be configured differently). If you can code on WordPress, try to follow the [wiki](https://github.com/dmuy/duDatepicker/wiki/Usage)
